### ISSUE TITLE
Issue with validating DateTimeFormatter.ofPattern("ddMMyyyy")

### ISSUE THREAD

**Issue by Istalan** [LGoodDatePicker/LGoodDatePicker#60]
Hi,

I have been using your DatePicker with great sucess so far, but i've encountered an issue with adding DateTimeFormatters without symbols between the d's, M's and y's to the FormatsForParsing. The String  does not get validated. I also added a few different Patterns and they work fine. I also looked at what happens if i use "ddMMyyyy" as FormatForDatesCommonEra on a DatePicker, that displays the current date and it displays it correctly but marks it as invalid (red Text).

My Code:

```
final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
final DatePickerSettings settings = new DatePickerSettings(Locale.GERMANY);
settings.setFormatForDatesCommonEra(formatter);
settings.setFormatForDatesBeforeCommonEra( DateTimeFormatter.ofPattern("d.M.yy.G"));
		
ArrayList parsingList = settings.getFormatsForParsing();
parsingList.add(DateTimeFormatter.ofPattern("d M yyyy"));
parsingList.add(DateTimeFormatter.ofPattern("d-M-yyyy"));
parsingList.add(DateTimeFormatter.ofPattern("ddMMyyyy"));
		
settings.setFormatsForParsing(parsingList);
//this part just as an example
DatePicker oneofmany = new DatePicker (settings.CopySettings());

```
I also checked, if  the standard Code:
`LocalDate date = LocalDate.parse( "02032017" ,DateTimeFormatter.ofPattern("ddMMyyyy"));`
works and it does.

I hope i communicated the Problem clearly, since neither an experienced programmer nor a native speaker. Other than this i found your package easy to work with and was very happy with the helpfull Documentation.

Thanks for all your hard work, even if you don't get around to this.

**Comment by BlakeTNC** [comment]
You are correct. I have written a function that fixes this. It turned out to be a pretty complicated problem to fix. I plan to commit the fix to the code base and if it does not break anything else, release it with the next release.

**Comment by BlakeTNC** [comment]
I fixed this issue and committed it to the repository. It will also be included in the latest release. Please let me know if it works for you now. Thanks for the bug report.

**Comment by BlakeTNC** [comment]
See the release 10.4.1.

**Comment by vindiagram** [comment]
This is not fixed in 10.4.1 it is still broken
